Libraries/Opc.Ua.PubSub/Encoding/JsonNetworkMessage.cs

```diff
@@ -147,6 +148,8 @@ public void SetNetworkMessageContentMask(JsonNetworkMessageContentMask networkMe
         {
             NetworkMessageContentMask = networkMessageContentMask;
 
+            // TODO: duplicate info for DataSetMessageContentMask..
```

Comment:
question is why does DataSetMessage needs to kknow the headercontentmask, it is only relevant if its encoded by networkmessage, so it can pas the setting. otherwise it needs to be tracked in two places, makes it error prone

Reply:
feel free to remove the comment if there is not an easy fix, but keep it for later